plugins/chat/app/models/chat/reviewable_message.rb

```diff
@@ -73,10 +73,17 @@ def build_actions(actions, guardian, args)
         )
       end
 
-      build_action(actions, :ignore, icon: "external-link-alt")
+      ignore_bundle =
+        actions.add_bundle(
+          "#{id}-ignore",
+          icon: "thumbs-up",
```

Comment:
Not sure why this has icon thumbs-up, because no icon is displayed. Copied this from the post reviewable. Will see if it can be removed. 🙂